config/cephlet-bucket/cephlet-rbac/cluster_role.yaml

```diff
@@ -177,15 +177,15 @@ rules:
       - update
       - watch
   - apiGroups:
-      - storage.api.onmetal.de
+      - storage.ironcore.dev
```

Comment:
Do we actually need those RBAC rules? Since the `ceph-provider` is not consuming any `ironcore` APIs anymore.

Reply:
removed